website/backend/settings.py

```diff
@@ -134,7 +134,7 @@
 
 # Static
 # Default to using webpack-dev-server on port 8081
-STATIC_HOST = config('WEB_STATIC_HOST', default='http://localhost:8081/')
+STATIC_HOST = config('REACT_WEB_STATIC_HOST', default='http://localhost:8081/')
```

Comment:
@123MwanjeMike hope you handled this environment variable on the config map

Reply:
Thanks @NoahNsimbe 
I've made the update